.flowconfig

```diff
@@ -6,6 +6,8 @@
 
 [options]
 
+experimental.strict_call_arity=true
```

Comment:
I had added this also to ensure that memoizeOne will work with the upcoming flow version